Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,511 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <GemCatalog/GemCatalogHeaderWidget.h>
+#include <CreateAGemScreen.h>
+#include <ProjectUtils.h>
+#include <PythonBindingsInterface.h>
+
+#include <QMessageBox>
+#include <QPushButton>
+#include <QStackedWidget>
+#include <QLabel>
+#include <QDialogButtonBox>
+
+
+namespace O3DE::ProjectManager
+{
+    CreateAGemScreen::CreateAGemScreen(QWidget* parent)
```

Comment:
Can we rename everything that is CreateAGemXYZ to CreateGem to be more consistent with our class names and qss names?